renderdoc/core/core.h

```diff
@@ -536,6 +536,12 @@ class RenderDoc
 
   string GetOverlayText(RDCDriver driver, uint32_t frameNumber, int flags);
 
+  void CycleActiveWindow();
+  uint32_t GetCapturableWindowCount()
+  {
+    return static_cast<uint32_t>(m_WindowFrameCapturers.size());
```

Comment:
Please prefer using a normal cast like `(uint32_t)m_WindowFrameCapturers.size()`, not `static_cast`.